src/clinical_etl/mappings.py

```diff
@@ -79,22 +91,31 @@ def date_interval(data_values):
 
     Args:
         data_values: a values dict with a date
+        date_format: string with order of day, month and year in date (e.g. "DMY", "YMD", etc.)
 
     Returns:
         A dictionary with calculated month_interval and optionally a day_interval depending on the specified
         date_resolution.
     """
     try:
         reference = INDEXED_DATA["data"]["CALCULATED"][IDENTIFIER]["REFERENCE_DATE"][0]
-    except Exception as e:
-        raise MappingError("No reference date found to calculate date_interval: is there a reference_date specified in the manifest?")
+    except KeyError:
+        raise MappingError("No reference date found to calculate date_interval: is there a reference_date specified in the manifest?", field_level=1)
+    DEFAULT_DATE_PARSER = dateparser.DateDataParser(
+        settings={"PREFER_DAY_OF_MONTH": "first", "DATE_ORDER": DATE_FORMAT}
```

Comment:
It might be good to declare DATE_FORMAT at the top of this file, like we do with INDEXED_DATA or IDENTIFIER.